Misc/NEWS.d/next/Windows/2022-05-16-11-45-06.gh-issue-92841.NQx107.rst

```diff
@@ -0,0 +1,2 @@
+:meth:`asyncio` no longer throws ``RuntimeError: Event loop is closed`` on
```

Comment:
```suggestion
:mod:`asyncio` no longer throws ``RuntimeError: Event loop is closed`` on
```

asyncio is a module, so needs the :mod: scope rather than the one for methods.

Reply:
Thank you, addressed.